src/Infrastructure/Repository/SettingRepository.php

```diff
@@ -101,9 +102,13 @@ public function getOneByPath(string $settingPath): SettingInterface
      */
     public function findProjectSettings(): array
     {
-        $settings = $this->findBy([
-            'isProject' => true,
-        ]);
+        $settings = $this->matching(
```

Comment:
I'd prefer using a queryBuilder for more complex queries in repositories for consistency

Reply:
Criteria is used for filtering collections (It's from doctrine collection package). In repository mostly should be used query builders for getting data.

Reply:
+